docs/news-widget/web-components/sesamy-content-container.md

```diff
@@ -98,12 +98,18 @@ When using the `lock-mode=signedUrl` lock mode the content is fetched from the s
 >
 ```
 
-### Unlock Message
+### Receipt Link
 
 This message is shown below the article once it is unlocked, it contains a link to the confirmation screen of the checkout where te item was purchased.
-It can be hidden by including the attribute `hide-unlock-message` in the sesamy-content-container tag (`<sesamy-content-container hide-unlock-message>`).
+It can be configured by setting the attribute `receipt-link` in the sesamy-content-container tag (`<sesamy-content-container receipt-link="none">`).
 
-The styling of the unlocke message can be configured with the following css variables:
+The following options are available fo the `receipt-link` attribute:
```

Comment:
`foR`  :wink

Reads like https://www.urbandictionary.com/define.php?term=Fo%20sure